### PR TITLE
Abstract awsh

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

**Review by tasodorff — CHANGES_REQUESTED** [review]
Two comments:
1. We need to define airwarshers, but it is unclear how they are different from makeup air units, fan coils, air handlers, pollution control units, etc.; what is an airwasher and how would someone distinguish it from another major type of equipment?
2. The tag AWSH doesnt exist in BDNS or in the building asset naming abbreviation list (v2.3): is there an existing acronym that fits this device?
3. The canonical types are not implemented properly. I expect complete canonicals, not a bunch of canonicals with a single abstract type implemented on them. Please fix.

**Comment by RathishKJ** [comment]
> Two comments:
> 
>     1. We need to define airwarshers, but it is unclear how they are different from makeup air units, fan coils, air handlers, pollution control units, etc.; what is an airwasher and how would someone distinguish it from another major type of equipment?
> 
>     2. The tag AWSH doesnt exist in BDNS or in the building asset naming abbreviation list (v2.3): is there an existing acronym that fits this device?
> 
>     3. The canonical types are not implemented properly. I expect complete canonicals, not a bunch of canonicals with a single abstract type implemented on them. Please fix.

1. Airwashers have been requested to have included in the BDNS  in the BDNS pull Number #112 . The airwasher implements MAU and the canonicals have been created around MAU.

2. Pull request has been created for inclusion of AWSH in the BDNS Pull Number 112

3. Complete canonicals have been created.

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml [hunk 1]
```diff
@@ -37,6 +37,12 @@ FAN:
   implements:
   - EQUIPMENT
 
+AWSH:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:40]: If we are modeling these at MAUs, then this is unnecessary. Revert.

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -1219,6 +1219,17 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+
+#telemetry for PLC#
+- controller_run_command:
```
Comment (tasodorff) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1225]: These are not used elsewhere in this PR. Please revert.

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 3]
```diff
@@ -1219,6 +1219,17 @@ literals:
...
+  - ON
+  - OFF
+  
+- controller_run_mode:
```
Comment (tasodorff) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1229]: Same

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 4]
```diff
@@ -752,4 +752,16 @@ MAU_US_SBO_1000_1:
...
+ 
+#Canonical for Air washers#
+MAU_SFSS_FDPM_SSPM_STM_SHM:
+  description: "makeup Air Unit with run command, status, dufferential pressure, static T,RH, monitoring and control"
+  implements:
+  - MAU
+  - SFSS
+  - FDPM
+  - SSPM
+  - STM
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:766]: How does it do this? DX or CHW?

**Review by tasodorff — CHANGES_REQUESTED** [review]
Some minor comments and clarifications.

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 4 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml:766]: The unit has water spray and doesnot have DX or CHW associated to the unit.

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml (hunk 1 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:40]: AWSH from general types has been removed...

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 2 above)
Comment (RathishKJ) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: Deleted from telemetry fields...

**Review by tasodorff — APPROVED** [review]

**Comment by tasodorff** [comment]
@RathishKJ approved once you fix errors

**Pull request merged**